src/query/server/server.go

```diff
@@ -162,6 +163,10 @@ func Run(runOpts RunOptions) {
 		logger.Fatal("could not create tag options", zap.Error(err))
 	}
 
+	if cfg.LookbackDuration == nil {
+		cfg.LookbackDuration = &defaultLookbackDuration
```

Comment:
This should be validated to be >= 0

Reply:
Good call

Reply:
nit: Might be better to put this check on a validate function, but not really a big deal

Reply:
Can you add this as a method on the config object?

i.e.
```go
func (c Configuration) LookbackDurationOrDefault() (time.Duration, error) {
  if c.LookbackDuration == nil {
    return DefaultLookbackDuration, nil
  }
  v := *c.LookbackDuration
  if v < 0 {
    return nil, anError
  }
  return v, nil
}
```